### DISCUSSION TITLE
Custom Logo

### DISCUSSION THREAD

**Discussion by John-H-Smith** [kromitgmbh/titra#168] [Ideas]
Hi there,
I would highly appreciate the option of uploading a custom logo instead of the titra default logo.
What are your opinions about that?

**Comment by faburem** [comment]
Should be quite easy to achieve in the navbar, but what about favicons and other stuff in the manifest.json? There are so many different image formats needed I don't think that is manageable to replace.
Would it be sufficient to just change the logo in the UI i.e. navbar?

**Reply by John-H-Smith** [reply]
I'm pretty sure it would.
Most open source applications only allow setting the logo and a favicon, though.

**Comment by faburem** [comment]
added it to our [roadmap](https://github.com/orgs/kromitgmbh/projects/2/views/1?pane=issue&itemId=23993901)

**Comment by faburem** [comment]
This feature is available in the latest version of titra! Go ahead and change the locks logo to your liking 🕺
